### DISCUSSION TITLE
I don't want the abstract capitalized. How to to that?

### DISCUSSION THREAD

**Discussion by gen-li** [alshedivat/al-folio#1490] [Q&A] [answered]
I don't want the abstract capitalized. How to to that?

**Comment by george-gca** [comment]
What you mean by abstract?

**Reply by gen-li** [reply]
The "ABSTRACT" button for selected publication section on the main page.

**Reply by george-gca** [reply, marked as answer]
Add to its style `text-transform: none;`
